Code/GraphMol/ChemReactions/ReactionParser.h

```diff
@@ -128,11 +129,25 @@ inline ChemicalReaction *RxnSmartsToChemicalReaction(
 }  // namespace v1
 //! returns the reaction SMARTS for a reaction
 RDKIT_CHEMREACTIONS_EXPORT std::string ChemicalReactionToRxnSmarts(
-    const ChemicalReaction &rxn);
+    const ChemicalReaction &rxn, const SmilesWriteParams &params);
+//! \overload
+inline std::string ChemicalReactionToRxnSmarts(const ChemicalReaction &rxn) {
+  SmilesWriteParams params;
+  params.canonical = false;
```

Comment:
This is going to take a while to go through, can you give me the nickel tour?  Why do we need canonical=False for instance?

Reply:
because it's SMARTS and we can't do sensible canonicalization with SMARTS